engine/src/main/java/org/pentaho/di/trans/TransMananger.java

```diff
@@ -0,0 +1,15 @@
+package org.pentaho.di.trans;
+
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.LogChannelInterface;
+
+import java.util.function.Supplier;
+
+public interface TransMananger  {
+  Trans createTrans( TransMeta transMeta, LogChannelInterface log, String runConfiguration,
```

Comment:
javadoc class comment for TransManager#create and all functions

Reply:
addressed